api/app/db/models/hfi_calc.py

```diff
@@ -78,12 +78,20 @@ class PlanningWeatherStation(Base):
     fuel_type_id = Column(Integer, ForeignKey('fuel_types.id'), nullable=False, index=True)
     planning_area_id = Column(Integer, ForeignKey('planning_areas.id'), nullable=False, index=True)
     order_of_appearance_in_planning_area_list = Column(Integer, nullable=True)
+    # Track which user created the record for auditing purposes.
+    create_user = Column(String, nullable=False)
+    create_timestamp = Column(TZTimeStamp, nullable=False, index=True)
```

Comment:
Do we need an index here? I don't think we sort / select on create_timestamp or update_timestamp anywhere do we?

Reply:
True, and we probably do want an index on `is_deleted` since that will be filtered on in most cases.